telegram/__init__.py

```diff
@@ -124,6 +123,7 @@
                                    SecureData,
                                    FileCredentials,
                                    TelegramDecryptionError)
+from .bot import Bot
```

Comment:
Unrelated?

Reply:
I think this was to avoid circular imports

Reply:
I think it was just moved a few lines but I guess that could avoid some rare circular import so I am fine with it :D